packages/liveblocks-react/src/room.tsx

```diff
@@ -264,7 +264,14 @@ function makeDeltaPoller_RoomThreads(client: OpaqueClient) {
   // Keep track of how many subscribers we've seen for every queryKey
   const countsByQuery = new Map<string, number>();
 
-  // XXX Stop using a queryKey here!
+  //
+  // XXX DISCUSSION
+  // XXX ----------
+  // XXX I _think_ we can remove the queryKey as a param here entirely and
+  // XXX "just" keep a flat count, like we do for our inbox notifications
+  // XXX poller. It would make the internals a lot simpler.
+  // XXX The query key isn't really used anyway.
+  //
```

Comment:
I do not recall why we did not keep a flat count to start with. But, in the current settings, I think you're right - we can keep a flat count instead and get rid of queryKey generation inside of the hook for good!!!

Reply:
Thanks for fixing this already 👍